### PR TITLE
Use stylelint-config-suitcss within preprocessor

### LINKED ISSUE #33: Preprocessor should use stylelint-config-suitcss by default
By doing so each component doesn't need to depend on `stylelint` and `stylelint-config-suitcss` and needs no configuration.

In user space the config can be extended or overridden by setting `options.stylelint`.

### PR DESCRIPTION
**Pull request by simonsmith**
Fixes #33

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -102,32 +102,23 @@ Where to resolve imports from. Passed to [`postcss-import`](https://github.com/p
 Ensure code conforms to the [SUIT code style](https://github.com/suitcss/suit/blob/master/doc/STYLE.md)
 by using the [stylelint-config-suitcss](https://github.com/suitcss/stylelint-config-suitcss) package.
 
-Stylelint [configuration options](http://stylelint.io/?/docs/user-guide/configuration.md) can also be overridden:
+Stylelint [configuration
```
Comment (giuseppeg) [inline README.md]: did you break here on purpose? does it work like that?

Comment (simonsmith) [inline README.md]: It does indeed. I wrap markdown at 80 cols. Can revert it though

Comment (giuseppeg) [inline README.md]: np

**Comment by giuseppeg** [comment]
:shipit:

**Pull request merged**